pkg/corerp/frontend/controller/applications/testdata/graph-app-httproute-out.json

```diff
@@ -13,18 +13,19 @@
         "type": "Applications.Core/containers"
     },
     {
+        "connections": [
+            {
+                "direction": "Inbound",
+                "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-app-ctnr"
+            }
+        ],
         "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Datastores/sqlDatabases/sql-db",
         "name": "sql-db",
         "provisioningState": "Succeeded",
         "type": "Applications.Datastores/sqlDatabases"
     },
     {
-        "connections": [
-            {
-                "direction": "Inbound",
-                "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-ctnr"
-            }
-        ],
+        "connections": [],
```

Comment:
This is correct because no other container resources add connection to http route.

Reply:
I'm not really going to comment on the correctness of the HTTP route scenario. It would take extra work specific to HTTP Routes to get a useful output.

We shouldn't fix it, we should complete the removal of the HTTP Route type.

Reply:
If we want to make it correct, then httproute should have outbound and inbound both.